### DISCUSSION TITLE
Flutter Resources

### DISCUSSION THREAD

**Discussion by CapillanesFrizty** [community/community#140042] [Programming Help]
### Body

Can anyone could help me in finding resources for flutter development?

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by habeshacoder** [comment]
you can use the official documentation. It is very clear and easy to follow and understand. Just create a new flutter project and code as you go on the doc:

you can start from here:
[https://docs.flutter.dev/get-started/install](url)
